apps/payment/models.py

```diff
@@ -553,3 +580,106 @@ def _send_receipt(self, payment_date, subject, description, payment_id, items, t
 
     class Meta:
         default_permissions = ('add', 'change', 'delete')
+
+
+class FikenSale(models.Model):
+    stripe_key = models.CharField(
+        'Stripe key',
+        max_length=20,
+        choices=StripeKey.ALL_CHOICES,
+        default=StripeKey.ARRKOM,
+    )
+    transaction_type = models.CharField(
+        'Transaction Type',
+        max_length=20,
+        choices=TransactionType.ALL_CHOICES,
+    )
+    original_amount = models.FloatField()
+    kind = models.CharField(
+        'Fiken Sale kind',
+        max_length=20,
+        choices=FikenSaleKind.ALL_CHOICES,
+        default=FikenSaleKind.CASH_SALE,
+    )
+    paid = models.BooleanField(default=True)
+    created_date = models.DateTimeField(default=timezone.now)
```

Comment:
I'd suggest using `auto_now_add`. That's a Django internal feature instead of setting the value with a function.

It's also marked as non-editable, so people can't just go in and edit it.

https://docs.djangoproject.com/en/2.2/ref/models/fields/#django.db.models.DateField.auto_now_add

Reply:
I considered it, and I can't really remember why, but I remember reading arguments against it.
I think I'll change it to `auto_now_add`